client/lib/plans/features-list.js

```diff
@@ -784,15 +784,15 @@ export const FEATURES_LIST = {
 	},
 	[ constants.FEATURE_ACTIVITY_LOG ]: {
 		getSlug: () => constants.FEATURE_ACTIVITY_LOG,
-		getTitle: () => i18n.translate( 'Expanded Site Activity' ),
+		getTitle: () => i18n.translate( 'Expanded site activity' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.